src/sentry/static/sentry/app/components/activity/noteInput.jsx

```diff
@@ -206,6 +231,21 @@ const NoteInput = React.createClass({
     }
   },
 
+  getMemberData() {
+    let sessionUser = ConfigStore.get('user');
+    let results = [];
+    this.state.memberList.forEach(member => {
+      if (sessionUser.id !== member.id) {
+        results.push({
+          id: member.id,
+          display: member.name,
+          email: member.email
+        });
+      }
+    });
```

Comment:
You could also use `filter` and `map` here. Not required, but it's a good exercise to get comfortable with them.